### PR TITLE
Minor spelling, punctuation, and consistency edits.

### PR DESCRIPTION
**Pull request by rkhoriander**

### CODE REVIEW

**Comment by rkhoriander** [comment]
https://auth0-docs-content-pr-6007.herokuapp.com/docs/tokens/access-token
https://auth0-docs-content-pr-6007.herokuapp.com/docs/connections/social/linkedin
https://auth0-docs-content-pr-6007.herokuapp.com/docs/identityproviders

File: articles/connections/social/linkedin.md [hunk 1]
```diff
@@ -8,67 +8,67 @@ description: This page shows you how to connect your Auth0 app to LinkedIn. You
...
-## 1. Log into the developer portal
+## 1. Log in to the Developer Portal
 
-Login to the [LinkedIn Developer portal](http://developer.linkedin.com/) and click **My Apps**:
+Log in to the [LinkedIn Developer portal](http://developer.linkedin.com/), and click **My Apps**:
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-1.png)
 
-## 2. Create your app
+## 2. Create Your App
```
Comment (mpaktiti) [inline articles/connections/social/linkedin.md]: We use title case for first-level headings (Connect Your App to LinkedIn) and sentence case for subheads (Create your app). See https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#formatting
We only capitalize stuff that are always capitalized, for example Lock.

I won't comment on every subhead in this PR so please go through them and undo these specific changes.

File: articles/connections/social/linkedin.md [hunk 2]
```diff
@@ -8,67 +8,67 @@ description: This page shows you how to connect your Auth0 app to LinkedIn. You
...
 
 `https://${account.namespace}/login/callback`
 
-Next click **Update**.
+Next, click **Update**.
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-4.png)
 
-## 5. Get your **Client ID** and **Client Secret**
+## 5. Get Your **Client ID** and **Client Secret**
```
Comment (mpaktiti) [inline articles/connections/social/linkedin.md]: No bold in subheads, the `## 5. Get Your **Client ID** and **Client Secret**` should be `## 5. Get your Client ID and Client Secret`

File: articles/tokens/access-token.md [hunk 3]
```diff
@@ -9,34 +9,34 @@ toc: true
...
 
-Auth0 currently generates Access Tokens in two formats: as opaque strings, or as [JSON Web Tokens (JWTs)](/jwt).
+Auth0 currently generates Access Tokens in two formats: as opaque strings or as [JSON Web Tokens (JWTs)](/jwt).
 
-This depends on the value that the **audience** parameter has in in the [authorization request](/api/authentication#authorize-client).
+This depends on the value that the **audience** parameter has in the [authorization request](/api/authentication#authorize-client).
 
-::: panel What is the audience?
-The **audience** is a parameter set during [authorization](/api/authentication#authorize-client), and it contains the unique identifier of the target API. This is how you tell Auth0 for which API to issue an Access Token, which is the intended *audience* of this token. If you do not want to access a custom API, then by setting the audience to `${account.namespace}/userinfo`, you can use the opaque Access Token to [retrieve the user's profile](/api/authentication#get-user-info).
+::: panel What is the Audience?
```
Comment (mpaktiti) [inline articles/tokens/access-token.md]: we don't capitalize audience

File: articles/tokens/access-token.md [hunk 4]
```diff
@@ -9,34 +9,34 @@ toc: true
...
 :::
 
 * If the **audience** is set to `${account.namespace}/userinfo`, then the Access Token will be an opaque string.
 * If the **audience** is set to the unique identifier of a custom API, then the Access Token will be a [JSON Web Token (JWT)](/jwt).
 
 When the **audience** is set to a custom API and the **scope** parameter includes the `openid` value, then the generated Access Token will be a JWT valid for both [retrieving the user's profile](/api/authentication#get-user-info) and for accessing the custom API. The **audience** parameter of this JWT will include two values: `${account.namespace}/userinfo` and your custom API's unique identifier.
 
-:::panel Use RS256 for multiple audiences
-If you set a custom API audience and also use `scope=openid` in your request, then your custom API must use **RS256** (read [how to change an API's settings](/apis#api-settings)). Tokens signed with HS256 can hold only one audience, for security reasons. This applies also if you have set a **Default Audience** at your [API Authorization settings](${manage_url}/#/tenant).
+:::panel Use RS256 for Multiple Audiences
```
Comment (mpaktiti) [inline articles/tokens/access-token.md]: no reason for capitalization, use sentence case at panels (except for things that are always capitalized, like Lock)

**Review by mpaktiti — CHANGES_REQUESTED** [review]

**Comment by mpaktiti** [comment]
@rkhoriander I left some comments. Follow the same process as before, once you push the changes remove the `update-needed` label and this will pop in my daily review list. Thank you!

**Review by mpaktiti — APPROVED** [review]

**Pull request merged**